<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilePropsDialog</class>
 <widget class="QWidget" name="ProfilePropsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>246</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame3">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>&amp;Profile name:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>editProfileName</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="editProfileName">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Short &amp;description:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>editDescription</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="editDescription">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel4_2">
        <property name="text">
         <string>&amp;Files in this profile will be owned by:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kurlInstallDir</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboUser"/>
      </item>
      <item>
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>&amp;Directory for this profile:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kurlInstallDir</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KUrlRequester" name="kurlInstallDir">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KLineEdit" name="labelInstallDir"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kcombobox.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
